### PR TITLE
feat(documentai): initial generation of documentai

### PR DESCRIPTION
**Pull request by busunkim96**

### CODE REVIEW

File: documentai/setup.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    author_email="googleapis-packages@google.com",
+    license="Apache 2.0",
+    url="https://github.com/GoogleCloudPlatform/google-cloud-python",
+    classifiers=[
+        release_status,
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
```
Comment (tseaver) [inline documentai/setup.py]: Do we want to be promising Python2 support for newly-generated APIs at this point?

Comment (busunkim96) [reply documentai/setup.py]: Probably not, removed

File: documentai/docs/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
+Python Client for Cloud Document AI API (`Alpha`_)
```
[the remark this answers is not in the document]
Comment (busunkim96) [reply documentai/docs/README.rst]: Done

**Comment by busunkim96** [comment]
Enabling Kokoro job internally

**Pull request merged**